src/Raven.Studio/typings/_studio/dto.d.ts

```diff
@@ -71,12 +71,30 @@ interface documentMetadataDto {
     '@flags'?: string;
     '@attachments'?: Array<documentAttachmentDto>;
     '@change-vector'?: string;
-
+    '@counters'?: Array<documentCounterDto>; // todo - not sure about adding this.. ???
 }
 
 interface updateDatabaseConfigurationsResult {
     RaftCommandIndex: number;
 }
+
+interface nodeCounterValue {
```

Comment:
it should be declared server side +server side should use nameof instead of string when writing response, see:
http://issues.hibernatingrhinos.com/issue/RavenDB-6016

I think it is worth doing this especially in new handlers.

Reply:
as discussed - there are server side classes - this is for studio use only